src/components/Button/styles.ts

```diff
@@ -0,0 +1,48 @@
+import styled from "styled-components";
+
+interface CommonProps {
+  // o prefixo $ é para evitar um warning do TypeScript
+  // ver mais: https://styled-components.com/docs/api#transient-props
+  $secondary?: boolean;
+}
```

Comment:
Aqui poderia adicionar um boolean para disabled, caso o botão passar disabled como true, adiciona os estilos de um botão disabled que tem no Figma e desativar o cursor, tipo `cursor: ${({ $disabled }) => $disabled ? 'not-allowed' : 'pointer'};`